bincapz.go

```diff
@@ -9,13 +9,15 @@ import (
 	"flag"
 	"fmt"
 	"io/fs"
+	"log"
```

Comment:
Rather than adding a second logging library, please re-use the `clog` import - even if, admittedly, I prefer the built-in Go library.

Reply:
Addressed in [`c978170` (#207)](https://github.com/chainguard-dev/bincapz/pull/207/commits/c978170042c5bc28de6d7c8a5739f7b2e4095e22).